pkg/health/client/modules_test.go

```diff
@@ -34,9 +35,7 @@ func TestGetAndFormatModulesHealth(t *testing.T) {
 		},
 		"happy-verbose": {
 			h: newTestMHappy(),
-			e: "Modules Health:\n" +
-				"  m1 OK status nominal 2s ago (x0)\n" +
-				"  m2 Degraded doh 20s ago (x0)",
+			e: "Modules Health:\nagent\n├── m1                                                      [OK] status nominal (2s, x0)\n└── a\n    └── b\n        └── c\n            ├── fred                                        [OK] yo (20s, x1)\n            │   └── blee                                    [OK] doh (20s, x1)\n            └── dork                                        [DEGRADED] bozo -- BOOM! (20s, x1)",
```

Comment:
Nit: using backticks here can help improve the readability of the expected output:

```go
			e: `Modules Health:
agent
├── m1                                                      [OK] status nominal (2s, x0)
└── a
    └── b
        └── c
            ├── fred                                        [OK] yo (20s, x1)
            │   └── blee                                    [OK] doh (20s, x1)
            └── dork                                        [DEGRADED] bozo -- BOOM! (20s, x1)`,
```